File: static/php/connection/DBConnection.php
<?php

/**
 * Created by PhpStorm.
 * User: Buddhi
 * Date: 8/19/2017
 * Time: 9:33 PM
 */
class DBConnection
{
    function getConnection()
    {
        $servername = "localhost";
        $username = "root";
        $password = "";
        $database = "adist";

        $conn = mysqli_connect($servername, $username, $password, $database);

        if ($conn->connect_error) {
            return $conn->connect_error;
        } else {
            return $conn;
        }

    }
}

File: app/db/DBConnection.py
import pymysql.cursors


class DBConnection():
    #get MySQL DB connection
    def getConnection(self):
        connection = pymysql.connect(host='localhost',
                                     user='root',
                                     password='',
                                     db='adist',
                                     charset='utf8mb4',
                                     cursorclass=pymysql.cursors.DictCursor)
        return connection
